CharlieBackend.Panel/Utils/ApiUtil.cs

```diff
@@ -18,6 +21,13 @@ public ApiUtil(IHttpUtil httpUtil)
             _httpUtil = httpUtil;
         }
 
+        private async Task EnsureSuccessStatusCode(HttpResponseMessage httpResponse)
+        {
+            string message = await _httpUtil.EnsureSuccessStatusCode(httpResponse);
+            if(message != string.Empty)
```

Comment:
Maybe IsNullOrEmpty would be better here? I know that this method returns string.empty, but still